docs/docs/features/rpc.mdx

```diff
@@ -42,6 +42,112 @@ Or with named params:
 
 The RPC server is configured using the `rpc` section on the configuration file, see the [configuration section](/running/config.mdx).
 
+## Health Check
+
+Health checks are available at the endpoint `health_nodeStatus` and retrieves the status of the node.
+
+### health_nodeStatus
+
+Retrieves information of the health of the node.
+
+#### Healthy node response
+
+<Tabs>
+  <TabItem value="list-param" label="Request">
+
+```json
+{
+	"jsonrpc": "2.0",
+	"id": 1,
+	"method": "health_nodeStatus",
+	"params": []
+}
+```
+
+  </TabItem>
+  <TabItem value="response" label="Response">
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "status": "Healthy",
+        "message": "Node running",
+        "syncingStatus": {
+            "starting_block_hash": "0x0336912d969aaffcd93f6e236e76ebcffbb1a73a7ae9f5c230ef010b29d3c014",
+            "starting_block_number": "b",
+            "current_block_hash": "0x0336912d969aaffcd93f6e236e76ebcffbb1a73a7ae9f5c230ef010b29d3c014",
+            "current_block_number": "b",
```

Comment:
Same as above

Reply:
is a hex value, direct response from the api